package.json

```diff
@@ -8,22 +8,21 @@
   },
   "scripts": {
     "prepare": "husky install && yarn build",
-    "clean": "lerna run clean && lerna clean --yes && rm -rf node_modules",
+    "clean": "nx run-many --target=clean --all && rm -rf node_modules/ .nx-cache/",
     "lint": "eslint packages/ scripts/ --ext=js,mjs,ts",
     "format": "prettier --write '{packages,scripts}/**/*.{js,mjs,ts,json,md}'",
     "bundlesize": "bundlesize --config ./scripts/bundlesize/bundlesize.config.json",
```

Comment:
I thought about running the `bundlesize` command with nx but decided against it because there doesn't seem to be an easy way to execute it in the context of nx.  

We need to use an nx [executor](https://nx.dev/packages/workspace/executors/run-commands) to run the command line call to `bundlesize` so that nx will recognize it as a build target (and build the dependencies before calling it).

It looks like the executor requires a `workspace.json` which [conflicts with the package.json/project.json files](https://github.com/nrwl/nx/issues/8874), meaning [nx will ignore the package.json](https://nx.dev/configuration/packagejson#workspace-json) when determining dependencies.